hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/transfer/Erc721TransferFromCall.java

```diff
@@ -61,8 +62,9 @@ public Erc721TransferFromCall(
             @NonNull final VerificationStrategy verificationStrategy,
             @NonNull final org.hyperledger.besu.datatypes.Address spender,
             @NonNull final HederaWorldUpdater.Enhancement enhancement,
+            @NonNull final SystemContractGasCalculator gasCalculator,
             @NonNull final AddressIdConverter addressIdConverter) {
-        super(enhancement);
+        super(gasCalculator, enhancement);
```

Comment:
Comment `// TODO - gas calculation` at L83 and as a result there are several `0L` gas charges in there.

Maybe you just haven't done gas charges for the transfers yet, that's coming?

Reply:
Fixed.